obs_status.yml

```diff
@@ -1331,7 +1332,7 @@ obs:
   status: 1
 - agasc_id:
   - 376724944
-  comments: Mag 6.5 spoiler 45 arcsec away
+  comments: Mag 6.5 spoiler 45 arcsec away. Set bad with bad position.
```

Comment:
same comment here: strictly speaking, if this is bad position, then it is not a spoiler, so we could remove that, right?